tests/lib/linter/linter.js

```diff
@@ -3408,6 +3425,52 @@ var a = "test2";
             });
         });
 
+        describe("physicalFilenames", () => {
+            it("should allow physicalFilename to be passed on options object", () => {
+                const physicalFilenameChecker = sinon.spy(context => {
+                    assert.strictEqual(context.getPhysicalFilename(), "foo.js");
+                    return {};
+                });
+
+                linter.defineRule("checker", physicalFilenameChecker);
+                linter.verify("foo;", { rules: { checker: "error" } }, { physicalFilename: "foo.js" });
+                assert(physicalFilenameChecker.calledOnce);
+            });
+
+            it("should allow physicalFilename to be passed as third argument", () => {
+                const physicalFilenameChecker = sinon.spy(context => {
+                    assert.strictEqual(context.getPhysicalFilename(), "foo.js");
+                    return {};
+                });
+
+                linter.defineRule("checker", physicalFilenameChecker);
+                linter.verify("foo;", { rules: { checker: "error" } }, { physicalFilename: "foo.js" });
+                assert(physicalFilenameChecker.calledOnce);
+            });
```

Comment:
This test looks to be identical to the previous one. It looks like the third argument case doesn't really apply to `physicalFilename` like it does to `filename`.

```suggestion
```

Reply:
Resolved